### PR TITLE
change deprecated ament_export_interfaces to ament_export_targets (foxy)

### PR DESCRIPTION
**Pull request by ddengster**

### CODE REVIEW

**Review by Yadunund — COMMENTED** [review]
Thanks for this change. I think the build action is failing due to eloquent sourced. Will update the action in a separate PR. Can I trouble you to update the [Changelog](https://github.com/osrf/rmf_schedule_visualizer/blob/master/rviz2_plugin/CHANGELOG.rst) to reflect this change?

**Comment by ddengster** [comment]
changelog updated, thanks @Yadunund

File: rviz2_plugin/CMakeLists.txt [hunk 1]
```diff
@@ -76,7 +76,7 @@ target_compile_definitions(${PROJECT_NAME} PRIVATE "RVIZ_DEFAULT_PLUGINS_BUILDIN
 pluginlib_export_plugin_description_file(rviz_common plugin_description.xml)
 
 ament_export_include_directories(include)
-ament_export_interfaces(${PROJECT_NAME} HAS_LIBRARY_TARGET)
+ament_export_targets(${PROJECT_NAME} HAS_LIBRARY_TARGET)
```
Comment (Yadunund) [inline rviz2_plugin/CMakeLists.txt]: Can we make this eloquent compatible too just like in https://github.com/osrf/rmf_demos/pull/126 ?

**Review by Yadunund — CHANGES_REQUESTED** [review]

File: rviz2_plugin/CMakeLists.txt (hunk 1 above)
Comment (ddengster) [reply rviz2_plugin/CMakeLists.txt]: 417f6bb

**Review by Yadunund — APPROVED** [review]
Thanks for addressing the issues! LGTM

**Pull request merged**